README.md

```diff
@@ -332,18 +352,18 @@ Set sleep params with worker pools `TypeBuilder` in both modules.
 2. Create your feature branch (`git checkout -b my-new-feature`)
 3. Commit your changes (`git commit -am 'Add some feature'`)
 4. Push to the branch (`git push origin my-new-feature`)
-5. Create new Pull Request
+5. Create a new Pull Request
 
 ### Running tests locally
 - Install diesel_cli.
 ```
 cargo install diesel_cli
 ```
-- Install docker in your machine.
+- Install docker on your machine.
 
 - Run a Postgres docker container. (See in Makefile.)
 ```
-make db
+create DB
```

Comment:
It is a Makefile command, why you change this ?

Also we can add a Justfile and use just that is a Rustcean `make`.

Reply:
```suggestion
create DB

You can run a docker container with PostgreSQL
```
make db
```
```

Reply:
Solved